src/main/java/de/hysky/skyblocker/config/configs/UIAndVisualsConfig.java

```diff
@@ -83,6 +86,36 @@ public static class ItemCooldown {
         public boolean enableItemCooldowns = true;
     }
 
+    public static class InventorySearchConfig {
+        @SerialEntry
+        public EnableState enabled = EnableState.SKYBLOCK;
+
+        @SerialEntry
+        public boolean ctrlK = false;
+
+        @SerialEntry
+        public boolean clickableText = false;
+
+        public enum EnableState {
+            OFF,
+            SKYBLOCK,
+            EVERYWHERE;
+
+            @Override
+            public String toString() {
+                return I18n.translate("skyblocker.config.uiAndVisuals.inventorySearch.state." + this.name());
+            }
+
+            public boolean isEnabled() {
+                return switch (this) {
+                    case OFF -> false;
+                    case SKYBLOCK -> de.hysky.skyblocker.utils.Utils.isOnSkyblock();
```

Comment:
uh why are we not importing the class? is there a name conflict or something?

Reply:
no. just felt like it